docker-compose.yml

```diff
@@ -41,6 +32,16 @@ services:
        - $CINDER_DIR/postgres:/var/lib/postgresql/data
```

Comment:
Should this be in the overrides YAML as well?

Reply:
Yes. I will update the PR.